ci/docker/buildenv/build-systemc.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+
```

Comment:
Please add to (most) every file a line (typically line 1 or 2) that describes the file, e.g.
      # DESCRIPTION: Builds SystemC components as part of making a docker image

Also probably should add a copyright, copy from elsewhere.